core/chains/evm/gas/models.go

```diff
@@ -289,7 +290,7 @@ func bumpGasPrice(cfg Config, lggr logger.SugaredLogger, currentGasPrice, origin
 	var priceByIncrement = new(big.Int)
 	priceByIncrement.Add(originalGasPrice, cfg.EvmGasBumpWei())
 
-	bumpedGasPrice := max(priceByPercentage, priceByIncrement)
+	bumpedGasPrice := bigmath.Max(priceByPercentage, priceByIncrement)
```

Comment:
Haven't see this one before. Could it be tightened up to accept only `big.Int`s now?
```go
func Max(x, y interface{})
```

Reply:
hm I think it could since it doesn't look like there are any heterogeneous argument uses, but that'd break consistency with the rest of these bigmath methods so probably best to look at in a followup if there's a cleaner way for all the bigmath helpers with generics instead of reflection.